### PR TITLE
title, thumbnail rendering etc

### PR DESCRIPTION
**Pull request by maboa**
this is the code I had lingering on my local disk, it enhances the UI slightly. please merge to develop.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -186,19 +236,34 @@ class App extends Component {
         <BackTop />
         <Content>
           <div style={{ background: '#fff', padding: 24, minHeight: 280 }}>
+
+          <div style={{ textAlign: 'right', paddingTop: 16, paddingBottom: 16 }}>
```
Comment (slifty) [inline src/App.js:240]: Should we be putting styles directly into the template, or would this be better done via CSS?

File: src/App.js [hunk 2]
```diff
@@ -186,19 +236,34 @@ class App extends Component {
...
+            <Switch checkedChildren="Simple" unCheckedChildren="Detailed" />
+          </div>
+
           <Search
             placeholder="please enter link here"
             size="large"
+            value="https://www.youtube.com/watch?v=4F4qzPbcFiA"
```
Comment (slifty) [inline src/App.js:247]: Is this hardcoded value for debug purposes?  Fine with keeping it for now but wanted to ask for the record.

**Review by slifty — COMMENTED** [review]
Given our sprint I'm OK with merging these as is, but if you could respond to the comments it would help guide future developments.

File: src/App.js (hunk 1 above)
Comment (maboa) [reply src/App.js:240]: Valid concern. I tend to prototype this way, at least while things are still in a state of flux. We can firm things up later.

File: src/App.js (hunk 2 above)
Comment (maboa) [reply src/App.js]: yup

Comment (maboa) [reply src/App.js:247]: yup

**Pull request merged**